### ISSUE TITLE
Wolfram|Alpha call returns no answer and chat cell fails.

### ISSUE THREAD

**Issue by rxm** [WolframResearch/Chatbook#905]
Describe the issue in detail here. Attach any relevant screenshots or files. The section below was automatically generated. Remove any information that you do not wish to include in the report.

<details>
<summary>Debug Data</summary>

| Property | Value |
| --- | --- |
| Name | ``"Wolfram/Chatbook"`` |
| Version | ``"1.4.6"`` |
| ReleaseID | ``"$RELEASE_ID$"`` |
| EvaluationEnvironment | ``"Session"`` |
| FrontEndVersion | ``"Desktop: 14.1 for Mac OS X x86 (64-bit) (July 16, 2024)"`` |
| KernelVersion | ``"14.1.0 for Mac OS X x86 (64-bit) (July 16, 2024)"`` |
| SystemID | ``"MacOSX-x86-64"`` |
| Notebooks | ``True`` |
| DynamicEvaluation | ``False`` |
| SynchronousEvaluation | ``False`` |
| TaskEvaluation | ``False`` |

## Settings

| Property | Value |
| --- | --- |
| StyleSheetVersion | ``"1.4.6.3927375719"`` |

## Stack Data
```
withChatState
conformToTextData
joinAdjacentStrings
makeInlineToolCallCell
openerView
wideScrollPane
throwFailure
trimURL
```

## Failure Data

```
<|
  "Evaluation" :> Wolfram`Chatbook`Tools`Private`formatWolframAlphaPods[Missing["NoResults"]["Query"], Missing["NoResults"]["Pods"], False]
  "Arguments" -> {}
  "Stack" :> Wolfram`Chatbook`Common`Private`$stack$514547
|>
```

</details>

**Comment by rxm** [comment]
Sorry about this one. I clicked on "Report this issue" and this ended up being created.  While evaluating a ChatInput, the system called Wolfram|Alpha, which returned no information (kind of expected given the question) and the error was labeled unexpected.    Having the evaluation fail seems like a reasonable solution, but maybe it should be labeled something other than "unexpected".

**Comment by rhennigan** [comment]
This was fixed in version 1.5.1. You can install the latest version of the Chatbook paclet with the following:
```
PacletInstall["Wolfram/Chatbook", UpdatePacletSites -> True]
```
